packages/server/src/api/routes/tests/search.spec.ts

```diff
@@ -2371,6 +2376,7 @@ describe.each([
       beforeAll(async () => {
         await config.api.application.addSampleData(config.appId!)
         table = DEFAULT_EMPLOYEE_TABLE_SCHEMA
+        rows = await config.api.row.fetch(table._id!)
```

Comment:
This is the one change that was required to make this test work with the in-memory search tests.